### DISCUSSION TITLE
@types/node: is @types/node@16 being developed?

### DISCUSSION THREAD

**Discussion by jjangga0214** [DefinitelyTyped/DefinitelyTyped#54225] [Issues with a @types package]
It seems no one created a PR, yet, though there are several changes in node v16([changelog](https://nodejs.org/blog/release/v16.0.0/), [community blog](https://medium.com/the-node-js-collection/node-js-16-available-now-7f5099a97e70)).

Is there who wants to collaborate?

Thanks.

**Comment by kentcdodds** [comment]
Looks like [`@types/node`](http://npm.im/@types/node) v16 was published 3 hours ago. I'm having trouble finding a changelog because it busted my global declarations 😅 https://twitter.com/kentcdodds/status/1411156097080053761
